changyunlee/src/components/Post.tsx

```diff
@@ -0,0 +1,214 @@
+import styled from "styled-components";
+import { ChangeEvent, FormEvent, useState } from "react";
+interface Props {
+  postImg: string;
+  profileImg: string;
+  user: string;
+  date: string;
+  postTitle: string;
+}
+interface Icomment {
+  id: number;
+  content: string;
+}
+const Post = ({ postImg, profileImg, user, date, postTitle }: Props) => {
+  const [count, setCount] = useState<number>(0);
+
+  const [comment, setComment] = useState<Icomment[]>([{ id: 0, content: "" }]);
+  const [currentComment, setCurrentComment] = useState<string>("");
+  const onCurrentCommentChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    setCurrentComment(value);
+  };
+  const onCommentSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setComment((prev) => [...prev, { id: 1, content: currentComment }]);
+    console.log(currentComment);
+    setCurrentComment("");
+  };
+
+  return (
+    <>
+      <StyledSection>
+        <StyledTitle>{postTitle}</StyledTitle>
+        <StyledDate>게시일: {date}</StyledDate>
+        <StyledPostImage src={postImg} alt="postImg" />
+        <StyledStatusCard>
+          <StyledA href="https://www.instagram.com/">
+            <StyledProfileRound>
+              <StyledProfileImg src={profileImg} alt="profileImg" />
+            </StyledProfileRound>
+            <StyledP>{user}</StyledP>
+          </StyledA>
+        </StyledStatusCard>
+        <StyledComment>
+          <span>comment...</span>
+          <StyledLike onClick={() => setCount(count + 1)}>
```

Comment:
```suggestion
          <StyledLike onClick={() => setCount(prev => prev + 1)}>
```

이렇게 비동기로 업데이트되는 걸 보장해 주면 좋을 거 같아요 ~